docs/quick-start/project-structure.md

```diff
@@ -0,0 +1,197 @@
+**Containers**
+A container is what they call a "Smart Component" in Redux. It is a component
+which knows about Redux. They are usually used as "Screens".
+
+Also located here are two special containers: `App.js` and `RootContainer.js`.
+
+`App.js` is first component loaded after `index.ios.js` or `index.android.js`. The purpose of this file is to setup Redux or any other non-visual "global" modules. Having Redux setup here helps with the hot-reloading process in React Native during development as it won't try to reload your sagas and reducers should your colors change (for example).
+
+`RootContainer.js` is the first visual component in the app. It is the ancestor of all other screens and components.
+
+You'll probably find you can get quite far in an Ignite app without even touching these two files. They, of course, belong to you, so when you're ready to add something non-visual, like Firebase, or something visual, like an overlay, you have places for these additions.
+
+**Containers ── Styles**
+This `/Containers/Styles` folder will house your container styles. Each container component will likely have a companion styles file, just like `/Components`.
+
+**Fixtures**
+TODO: Correct this, the description is outdated.
+
+All key API responses are housed here.
+
+These API responses can be used for several reasons.  _E.G._:
+* To bypass logins when building any screen of the application
+* To quickly test API parsing in unit tests
+* To separate Network from Data concerns while coding
+
+**Images**
+Static images used in your project are stored here.
+
+**Lib**
+At first glance, this could appear to be a "miscellaneous" folder, but we recommend that you treat this as proving ground for components that could be reusable outside your project.
+
+Maybe you're writing a set of utilities that you could use outside your project, but they're not quite ready or battle tested. This folder would be a great place to put them. They would ideally be pure functions and have no dependencies related to other things in your App folder.
+
+**Navigation**
+TODO: Update with information about new navigation.
+
+**Redux**
+A place to store your Redux files (reducers, stores, etc.).
+
+**Sagas**
+A place to store your Sagas (Redux side effects).
+
+**Services**
+API calls to external services?
+
+**Themes**
+A place to contain styles shared across you project (fonts, colors, etc.).
+
+**Transforms**
+A common pattern when working with APIs is to change data so that it plays nice between your app & the API.
+
+We've found this to be the case in every project we've worked on. So much so that we're recommending that you use this folder for these transformations.
+
+Transforms are not necessarily a bad thing (although an API might have you transforming more than you'd like).
+
+For example, you may:
+
+* turn appropriate strings to date objects
+* convert snake case to camel case
+* normalize or denormalize things
+* create lookup tables
+
+## Ignite directory
+
+TODO:
```

Comment:
These are your dev screens and examples that appears in your app, but we recommend not shipping your product with this.

Generators won't be here though.  It's technically not all things ignite.  It's all the visual parts of ignite.  The generators & plugins live in `/node_modules` and `/ignite/plugins`.  😃